### PR TITLE
Feature/tp/add speed bumps #448

### PR DESCRIPTION
**Pull request by ThomasSchloemicherAVL**
﻿#### Reference to a related issue in the repository
Add a reference to a related issue in the repository.

#### Add a description
Add a description of the changes proposed in the pull request.

**Some questions to ask**:
What is this change?
What does it fix?
Is this a bug fix or a feature? Does it break any existing functionality or force me to update to a new version?
How has it been tested?

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [x] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html).
- [ ] I have taken care about the [documentation](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/commenting.html).
- [x] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [x] My changes generate no errors when passing CI tests. 
- [ ] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

If you can’t check all of them, please explain why.
If all boxes are checked or commented and you have achieved at least one positive review, you can assign the label ReadyForCCBReview!

### CODE REVIEW

**Comment by ThomasSchloemicherAVL** [comment]
Renaming of branch done to stay compliant with branching naming convention

**Comment by pmai** [comment]
I think this needs to be rebased on current master, since it includes lots of Wheel Data content which might conflict.

Other than that I think this looks good to me...

File: osi_object.proto [hunk 1]
```diff
@@ -61,6 +61,22 @@ message StationaryObject
...
+        //
+        repeated Identifier assigned_lane_id = 5;
+
+        // Percentage value of the total stationary object residing within a corresponding lane.
+        //
+        // \note Might be multiple if the stationary object is streching over multiple lanes.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated double assigned_lane_percentage = 6;
```
Comment (ThomasNaderBMW) [inline osi_object.proto:78]: Can't this be calculated out of base in comparison to lane width?
Or did I miss the point here? => Then a better description would be good

Comment (kmeids) [reply osi_object.proto:78]: Output from CCB - 03.03.2021:
1. Move _repeated Identifier assigned_lane_id_  and _repeated double assigned_lane_percentage_ to another pull request and consider adding them to a new "Static Object Classification" (see Moving Object Classification)

**Comment by kmeids** [comment]
Output from CCB 03.03.2021:
1. Only **TYPE_SPEED_BUMP = 17** will be considered in this pull request.
2. @pmai will merge this to master.

**Comment by pmai** [comment]
Will be merged to master once 3.3.0 is cut, i.e. after 2021-03-17 TSC decision.

**Review by pmai — APPROVED** [review]

**Comment by kmeids** [comment]
** Output of CCB 12.05.2021**:
1. @pmai to do the merge after switching to  Github actions.

**Pull request merged**